content/using/develop.md

```diff
@@ -6,7 +6,11 @@ template = "page_indiced.html"
 aliases = ["/docs/using/creating-a-development-ship/", "/docs/getting-started/contributing"]
 +++
 
-There are roughly two kinds of development on Urbit: core development and application development. Urbit applications are still young, but can be fun to experiment with. The kernel is much more stable, but generally more challenging from an engineering standpoint.
+You can develop on Urbit in three ways:
+
+- [Applications using Urbit](#api), which use Urbit as the back-end to store data and communicate with others
+- [Urbit Applications](#application), which are written in [Hoon](/docs/glossary/hoon/), and run as part of Urbit
```

Comment:
@jalehman Did we decide to say "Urbit Agents" instead?

Reply:
Can't remember if we decided on that or not. I like the idea, but it's not exactly correct. Technically they're *gall* agents, but I worry about adding yet another term into the mix here. Perhaps if there was a natural way to link to the correct concept—gall agents—the white lie could be both (mostly) correct and newcomer-friendly?